packages/ffe-form/less/tooltip.less

```diff
@@ -62,4 +62,34 @@
         overflow: visible;
         visibility: visible;
     }
+
+    &--dark {
+        &__icon {
+            background-color: transparent;
+            border: 2px solid @ffe-grey-silver;
+            color: @ffe-blue-focus;
+
+            &:hover {
+                border-color: @ffe-white;
+                //color: @ffe-grey-gunmetal--dark;
```

Comment:
Any reason this is commented out? Same thing on line 89.

Reply:
My bad. We did some last minute testing to get rid of a color and forgot to remove the comment with the old color. I'll fix it right away.